app/styles/_state.scss

```diff
@@ -10,11 +10,17 @@
   color: $warning;
 }
 
+.is-triggered {
+  background: linear-gradient(to bottom, #060606 0%, #2d2d2d 100%);
```

Comment:
@pdehaan: let's leave these be for now, i think color management is going to require a bit more thought.